docker-compose.yml

```diff
@@ -9,10 +9,17 @@ services:
     build: ./
     env_file:
       - .env
-    volumes:
-      - "./autogpt:/app"
-      - ".env:/app/.env"
+    # Only uncomment this if you really want to expose the host directory to container via bind mount (less safe)
```

Comment:
This was an intentional decision and shouldn't be undone. Without it, users need to rebuild every single time they make a code change which is impractical. The safety benefits are marginal but this makes using the docker setup significantly more painful.

Reply:
I see your point and actually I usually prefer to bind mount source to the container as it eliminates the need to docker cp files into or out of the container and also simplifies revision control and .vscode setup. In this case, safety was the overriding concern.

Your changes in #1199 are nicely done (relocating source code to /home/appuser/app) and I would opt for that one now.

Reply:
btw I really like the change to `redis-stack` so definitely keep that change in here and maybe just rename the PR to focus on that